addons/acre/functions/fnc_adjustVoiceVolume.sqf

```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ *
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+
+if (!hasInterface) exitWith { nil };
+
+private _volumeLevel = [GVAR(startBppVolume), GVAR(startLeaderVolume)] select (player isEqualTo leader player);
+
+private _volume = switch (round _volumeLevel) do {
```

Comment:
What do you think about making this setting a dropdown instead of rounding the slider number?

Reply:
I thought about it when doing that but I could not find any translations for speech volume levels in ACRE. But yes, we can do it ourself and this should be more user friendly.